Runtime/ExtensionMethods.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine.Networking;
+
+namespace BlockadeLabsSDK
+{
+    internal static class AwaiterExtensions
```

Comment:
why move this?

Reply:
I prefer all classes/members and types to be in their own files per MSDN